app/data/test_navigation.json

```diff
@@ -0,0 +1,667 @@
+{
```

Comment:
Maybe, its sounds like a high level discussion/retrospective and not for this time or story. We need to get this in otherwise we are going to miss the deadline, is everything test? Yes.